src/pages/Home/Home.layout.jsx

```diff
@@ -16,13 +17,24 @@ import { Button, Header, Logo, ManImage, Text } from './Home.styled'
  */
 const Home = ({ title, subtitle, buttonText }) => (
   <Header>
-    <Logo src={logo} alt="logo" />
-    <Text as="h1" color="secondary" large>
-      {title}
-    </Text>
-    <Text small>{subtitle}</Text>
-    <ManImage src={deliveryMan} alt="Delivery man" />
-    <Button to="/check">{buttonText}</Button>
+    <HalfSection bg={Colors.yellow06}>
+      {/**/}
```

Comment:
These spaces are not necessary, can you try to remove them?

Reply:
ready